api/database_snapshot_types.go

```diff
@@ -25,21 +25,21 @@ type DatabaseSnapshotSpec struct {
 	SnapshotSpec `json:",inline,omitempty"`
 }
 
-type SnapshotStatus string
+type SnapshotPhase string
 
 const (
 	// used for DatabaseSnapshots that are currently running
-	StatusSnapshotRunning SnapshotStatus = "Running"
+	PhaseSnapshotRunning SnapshotPhase = "Running"
```

Comment:
I think `SnapshotPhaseRunning` is better. All the phase values with same prefix can be declared in the same `const()` block.